bundle/bundle.go

```diff
@@ -165,12 +165,12 @@ func Read(path string) (*Bundle, error) {
 	// first, find the Directive.
 	for _, f := range r.File {
 		if f.Name == "Directive.yaml" {
-			directive, err := readDirective(f)
+			Reading_directive, err := readDirective(f)
```

Comment:
```suggestion
			d, err := readDirective(f)
```
In this case let's just do the simple thing and use `d`, as we do in other places in this file:

https://github.com/suborbital/atmo/blob/5b27fc4afc02ee344f36b0c63f8756c7c9ca66d9/bundle/bundle.go#L234-L239

I think it's perfectly fine to use a short name here because it is only used within a single scope.

Note: for identifier names in Go, [the convention](https://go.dev/doc/effective_go#mixed-caps) is to use `mixedCaps` for multi-word identifiers.